src/components/FlowRunLogs.vue

```diff
@@ -11,14 +11,14 @@
             <div v-if="logLevel > 0">
               No logs match your filter criteria
             </div>
-            <div v-else-if="flowRun.stateType == 'scheduled'">
+            <div v-else-if="flowRun.stateType == 'scheduled'" class="flow-run-logs__empty-text">
               This run is scheduled and hasn't generated logs
             </div>
-            <div v-else-if="flowRun.stateType == 'running'">
+            <div v-else-if="!isTerminalStateType(flowRun.stateType)" class="flow-run-logs__empty-text">
               Waiting for logs...
             </div>
-            <div v-else>
-              This run didn't generate Logs
+            <div v-else class="flow-run-logs__empty-text">
+              This run didn't generate logs
```

Comment:
Can we just add the class directly to `p-empty-results` rather than on each individual message? I'm guessing the "No logs match your filter criteria" also need the white test.

Reply:
<img width="540" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/40272060/27bfdec6-a1ea-469f-9d24-b75602947e06">

Reply:
Sorry adding photo so don't lose it. 

The p-empty-results has an inner div that seems to conflict with whatever I set on the p-empty-results in the log container. Adding to the internal divs gives better targeting.  But good call on "No logs match your.."

Reply:
Hmm we should fix that in prefect design. But until that can we wrap all the slot text in a single div and add the class to that? So we don't keep repeating the class